src/routes/dashboard/+page.svelte

```diff
@@ -30,10 +30,10 @@
 					<TableBodyCell>{project.name}</TableBodyCell>
 					<TableBodyCell>{project.fqdn}</TableBodyCell>
 					<TableBodyCell class="whitespace-pre-line">
-						{project.owner.email} ({$i18n.t('dashboard.owner')})
 						{project.users
 							.map(
-								(u) => `${u.profile.email} ${u.is_admin ? `(${$i18n.t('dashboard.admin')})` : ''}`
+								(u) =>
+									`${u.profile.email} ${u.admin_level == 'owner' ? `(${$i18n.t('dashboard.owner')})` : `(${$i18n.t('dashboard.admin')})`}`
```

Comment:
```suggestion
								(u) =>
									`${u.profile.email} ${u.admin_level === 'owner' ? `(${$i18n.t('dashboard.owner')})` : `(${$i18n.t('dashboard.admin')})`}`
```

stp de manière générale : met toujours le strict equality operator